chrome/browser/net/chrome_network_delegate.cc

```diff
@@ -327,6 +330,95 @@ int ChromeNetworkDelegate::OnBeforeURLRequest(
   return rv;
 }
 
+std::string generateBlockedElemhideJavaScript(std::string url) {
+  std::stringstream ss;
+
+  // basically copy-pasting JS from
+  // https://github.com/adblockplus/adblockpluschrome/blob/master/include.preload.js#L546
+  ss <<
+"function hideElement(element)\n\
```

Comment:
It can be easier with C++ raw strings. Could you please change it, if there is no easy way with resources yet.

Reply:
Yeah, we can discuss it separately. Meanwhile, it seems we can just return a raw string.
BTW, it would be better to use streams to set the URL or even just replace some template entry in the string. E.g. https://github.com/adblockplus/adblockplusie/blob/05f8b81085d9da9ae03d7e5ddd51ba47c491d01d/src/shared/Utils.cpp#L174 (pay attention that it's ABP for IE project).